crates/ty_python_semantic/resources/mdtest/attributes.md

```diff
@@ -399,9 +395,19 @@ class TupleIterable:
 
 class C:
     def __init__(self) -> None:
+        # TODO: Should not emit this diagnostic
+        # error: [unresolved-attribute]
```

Comment:
There are around [1600 new `unresolved-attribute` diagnostics](https://shark.fish/type-of-self-diff.html). Is it plausible that some of them are related to this?